j2eeapis/nbproject/project.properties

```diff
@@ -16,4 +16,4 @@
 # under the License.
 
 is.autoload=true
-release.external/jsr88javax.jar=modules/ext/jsr88javax.jar
+release.external/deployment-api-1.2-rev-1.jar=modules/ext/deployment-api-1.2-rev-1.jar
```

Comment:
I would prefer:
```
release.external/deployment-api-1.2-rev-1.jar=modules/ext/jsr88javax.jar
```
That reduces the necessary changes to other modules and third parties will be unaffected. This would still be valid, as the new jar is still JSR88.